tests/test_abi.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from __future__ import division
+from builtins import zip
+from builtins import range
+from builtins import object
+from past.utils import old_div
```

Comment:
Do not import `past` please, there are two division operators: `/` and `//`, see:

```
[dc@dc:~]$ python3 -c 'from __future__ import division, print_function; print(1/2, 1//2)'
0.5 0
[dc@dc:~]$ python2 -c 'from __future__ import division, print_function; print(1/2, 1//2)'
0.5 0
```